tests/end_to_end/features/steps/filestore_upload.py

```diff
@@ -0,0 +1,195 @@
+from behave import *
```

Comment:
for this file, does file stores not support uploading a file? If not, then this is fine. And maybe when/if we support it then we can change this to use that API.

Reply:
Yeah that was the goal